pkg/networkservice/common/clienturl/client.go

```diff
@@ -46,9 +48,10 @@ type clientURLClient struct {
 // NewClient - creates a Client that will using clienturl.ClientUrl(ctx) to extract a url, dial it to a cc, use that cc with the clientFactory to produce a new
 //             client to which it passes through any Request or Close calls
 // 	ctx	- is full lifecycle context, any started clients will be terminated by this context done.
-func NewClient(ctx context.Context, clientFactory client.Factory, dialOptions ...grpc.DialOption) networkservice.NetworkServiceClient {
+func NewClient(ctx context.Context, dialTimeout time.Duration, clientFactory client.Factory, dialOptions ...grpc.DialOption) networkservice.NetworkServiceClient {
```

Comment:
Why it is not an option?

Reply:
Moved `clienturl` client to `connect` and made it private.